### PR TITLE
Feature/remove geometry intersection shader

### PR DESCRIPTION
**Pull request by andybak**
This removes all the clever stuff and does something mind-boggling simple instead. I will be very surprised if this fix works.

### CODE REVIEW

**Comment by andybak** [comment]
Note for testing - if you want to simulate running on a Quest 1 (and therefore use the new code path) set GeometryShaderSuppported to always return false.

Last time I tested, there was a major crash loading some scenes when using this code path.

File: Assets/Resources/Brushes/Shared/Shaders/Special/Intersection.shader [hunk 1]
```diff
@@ -171,7 +171,7 @@ Shader "Brush/Special/Intersection" {
           //&& any(input[0].worldPos != input[1].worldPos)
           //&& any(input[0].worldPos != input[2].worldPos)
           //&& any(input[2].worldPos != input[1].worldPos);
-
```
Comment (mikeskydev) [inline Assets/Resources/Brushes/Shared/Shaders/Special/Intersection.shader:174]: No need for this!

File: Assets/Resources/Brushes/Shared/Shaders/Special/IntersectionNonGeomFallback.shader [hunk 2]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 The Tilt Brush Authors
```
Comment (mikeskydev) [inline Assets/Resources/Brushes/Shared/Shaders/Special/IntersectionNonGeomFallback.shader:1]: Year, attribution change

**Review by mikeskydev — CHANGES_REQUESTED** [review]
Works on Quest 1. Nice stuff! Just a few nits.

**Pull request merged**